dev/arkdark/.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+// A launch configuration that launches the extension inside a new window
```

Comment:
Could we move this to the repo root and still have it open from `./dev/arkdark`?